lib/rspec/core/hooks.rb

```diff
@@ -13,13 +13,14 @@ module Hooks
       # @overload before(scope, &block)
       #   @param scope [Symbol] `:example`, `:context`, or `:suite`
       #     (defaults to `:example`)
-      # @overload before(scope, conditions, &block)
+      # @overload before(scope, *conditions, &block)
```

Comment:
I'm still unsure about the changes in all this file, can you show some sample renders before / after? They imply the main use case is an array of symbols with a hash as an afterthought when really we expect this to be a hash 99% of the time.

Reply:
Sure.

Before:
![image](https://user-images.githubusercontent.com/6916/72385265-2829b000-3730-11ea-9b46-d145304ce7ad.png)

After:
![image](https://user-images.githubusercontent.com/6916/72385067-c1a49200-372f-11ea-8a2f-f8f611099342.png)

The difference besides the signature in the second overload is the changed `conditions` description.